src/components/Movie.js

```diff
@@ -0,0 +1,17 @@
+import '../css/pulpitAndMovie.scss';
```

Comment:
usually you just create "main.scss" file where you import the rest off scss files and then you import main.scss in the index js file of your react project
![image](https://user-images.githubusercontent.com/17354422/133222935-aacf5a86-8552-442b-869d-0c1953fb177a.png)